src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -127,13 +127,15 @@ def test_set_read_dev_agreement_error(self):
             assert doc(selector).text() == 'This field is required.'
 
     def test_read_dev_agreement_skip(self):
+        set_config('last_dev_agreement_change_date', '2019-06-10 00:00')
         after_agreement_last_changed = (
             datetime(2019, 6, 10) + timedelta(days=1))
         self.user.update(read_dev_agreement=after_agreement_last_changed)
         response = self.client.get(reverse('devhub.submit.agreement'))
         self.assert3xx(response, reverse('devhub.submit.distribution'))
 
     def test_read_dev_agreement_set_to_future(self):
+        settings.DEV_AGREEMENT_CHANGE_FALLBACK = datetime(2019, 6, 10, 12, 00)
```

Comment:
Use `override_settings` decorator to alter settings in tests.

```python
    @override_settings(DEV_AGREEMENT_CHANGE_FALLBACK=datetime(2019, 6, 10, 12, 00))
    def ...   
```

Reply:
7f91a2559f